README.rst

```diff
@@ -75,6 +75,7 @@ What's New?
 
 in development
 ^^^^^^^^^^^^^^
+* Fix broken option ``--prepend-package`` in pydoctor 22.7.0.
```

Comment:
maybe a bit of more info for the release notes.

```suggestion
* The ``--prepend-package`` now work as documented and DOES-EXPECTED-BEHAVIOUR. A regression was introduced in pydoctor 22.7.0 and it was DOING-UNEXPECTED-BEHAVIOUR.
```

Reply:
I think that this was not applied... can be applied as a drive-by for another PR .
No problem.